aviatrix/utils.go

```diff
@@ -298,6 +299,11 @@ func checkPAVMVersionFormat(version string) string {
 
 // compareVersion compares two Semantic Versions
 func compareVersion(version1, version2 string) bool {
+	defer func() {
```

Comment:
You probably don't need to defer here. `version.NewVersion` returns an error as its second return value. Right now you have not assigned it to anything. You can do something like this:
```
v1, err := version.NewVersion(version1)
if err != nil {
  // Log if needed
  return false
}
```

Reply:
that cash error comes from the v1.GreaterThan(v2) function and I check that function doesn't have the error as its second return value.

Reply:
have fixed